app/upgrades/v16/upgrades.go

```diff
@@ -77,6 +85,39 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// Specifying the whole list instead of adding and removing. Less fragile.
```

Comment:
I think that retrieving from params and appending new items is safer instead of having the risk of something slipping through from this large list during review or on subsequent upgrade handler updates.

Reply:
same opinion here as well, (suppose a situation where we had a code freeze and have a planned date for update, but params got updated via governance in between)

Reply:
done here https://github.com/osmosis-labs/osmosis/pull/5380/commits/23615fdf6c2fb28e5d9400e08e7776bf6330c968